kong/db/schema/others/declarative_config.lua

```diff
@@ -830,6 +838,20 @@ local function flatten(self, input)
         end
       end
 
+      if schema.ttl then
+        local ttl_value = entry.ttl
+        if ttl_value and ttl_value ~= 0 and ttl_value ~= null then
+          if entry.updated_at then
+              ttl_value = ttl_value + entry.updated_at
+          elseif entry.created_at then
+              ttl_value = ttl_value + entry.created_at
+          else
+              ttl_value = ttl_value + now_updated()
+          end
+          flat_entry.ttl = ttl_value
```

Comment:
The name is not accurate. TTL should be a duration, while `expired_at` could be used here as a certain moment.

Reply:
done